src/groups/mqb/mqbstat/mqbstat_queuestats.cpp

```diff
@@ -609,25 +585,27 @@ void QueueStatsDomain::onEvent(EventType::Enum    type,
 
     BALL_LOG_SET_CATEGORY(k_LOG_CATEGORY);
 
-    if (!d_subContexts_mp) {
+    if (d_subContextsLookup.empty()) {
         BALL_LOGTHROTTLE_WARN(k_MAX_INSTANT_MESSAGES, k_NS_PER_MESSAGE)
-            << "[THROTTLED] No built sub contexts";
+            << "[THROTTLED] No built sub contexts for domain: "
+            << d_statContext_mp->name() << ", appId: " << appId;
```

Comment:
New log line looks like
`06AUG2024_02:21:59.442 10822:8281000960 WARN /Users/emalygin/work/blazingmq/src/groups/mqb/mqbstat/mqbstat_queuestats.cpp:602 MQBSTAT.QUEUESTATS [THROTTLED] No matching StatContext for domain: bmq://mock-domain/abc, appId: bar`